src/plugins/janus_sip.c

```diff
@@ -6625,14 +6652,27 @@ static int janus_sip_allocate_local_ports(janus_sip_session *session, gboolean u
 			session->media.pipefd[1] = -1;
 		}
 	}
+	/* Determine the address family to use */
+	if(ipv6_disabled &&
+		!janus_network_address_is_null(&janus_network_local_media_ip) &&
+		janus_network_local_media_ip.family == AF_INET6) {
```

Comment:
Code style: you need another tab of indent for the two additional checks above, otherwise it ends up lined with the action below.

Reply:
Point taken. The condition is removed as redundant.